srcpkgs/spago/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'spago'
+pkgname=spago
+version=0.16.0
+revision=1
+build_style="haskell-stack"
+hostmakedepends="stack"
+makedepends="ncurses-libtinfo-devel tar xz zlib-devel"
```

Comment:
I thought the point of your `stack.yaml` patch was to drop the dependence on `ncurses-libtinfo-devel`.

Reply:
The patch is to ensure that `libtinfo.so.5` isn't used during the build process of spago.
There is a dependency called `haskeline` that requires it, which the patch addresses.

`libtinfo.so.6` is required so the correct version of `ghc` can be downloaded, which is what `ncurses-libtinfo-devel` installs via symlinks.